test/extended/images/performance.go

```diff
@@ -0,0 +1,46 @@
+package images
```

Comment:
This directory seems to be more about ImageStreams and such.  CRI-O pull speed (which is what these tests are about) seems like it deserves a different maintainer set.  Sibling `test/extended/node` or some such?